### PR TITLE
JDF-235 - Shift Aerogear Quickstarts to JDF

### PR DESCRIPTION
**Pull request by rafabene**
I also removed CONTRIBUTING.md because I consider that from now users should use the QUICKSTARTS_ROOT/CONTRIBUTING.md and also removed Origin tag from READMEs.

### CODE REVIEW

File: helloworld-html5/README.md [hunk 1]
```diff
@@ -5,7 +5,6 @@ Level: Beginner
 Technologies: CDI, JAX-RS, HTML5
 Summary: Basic HTML5 |Demonstrates the use of CDI 1.0 and JAX-RS using the HTML5 architecture and RESTful services on the backend
 Target Product: WFK
-Source: <https://github.com/aerogear/as-quickstarts>
```
Comment (sgilda) [inline helloworld-html5/README.md:8]: This still needs a 'Source' tag.
`Source: <https://github.com/jboss-jdf/jboss-as-quickstart>`

File: kitchensink-cordova/README.md [hunk 2]
```diff
@@ -1,16 +1,15 @@
-kitchensink-cordova: Example AeroGear Application Using Hybrid HTML5 + REST with Apache Cordova
+kitchensink-cordova: Example Mobile Application Using Hybrid HTML5 + REST with Apache Cordova
 ===============================================================================================
 Author: Kris Borchers
 Level: Intermediate
 Technologies: HTML5, REST, Apache Cordova
 Summary: Based on kitchensink, but uses hybrid HTML5 running as a native application on mobiles
 Target Product: WFK
-Source: <https://github.com/aerogear/as-quickstarts>
```
Comment (sgilda) [inline kitchensink-cordova/README.md:8]: Again, needs the 'Source' tag
`Source: <https://github.com/jboss-jdf/jboss-as-quickstart>`

File: kitchensink-html5-mobile/README.md [hunk 3]
```diff
@@ -1,11 +1,10 @@
...
+kitchensink-html5-mobile: Example Application Using Multiple HTML5, Mobile & JAX-RS Technologies 
 =========================================================================================================
 Author: Jay Balunas
-Level: Intermediate
-Technologies: Aerogear, CDI, HTML5, REST
+Level: Beginner 
+Technologies: CDI, HTML5, REST
 Summary: Based on kitchensink, but uses HTML5, making it suitable for mobile and tablet computers
 Target Product: WFK
-Source: <https://github.com/aerogear/as-quickstarts>
```
Comment (sgilda) [inline kitchensink-html5-mobile/README.md:8]: Add:
`Source: <https://github.com/jboss-jdf/jboss-as-quickstart>`

**Comment by sgilda** [comment]
Other than the missing 'Source' tags, this looks good to me. We can always add those after the merge.

**Comment by rafabene** [comment]
Squashed Merge

**Pull request closed without merging**